static_home_page/templates/home_page/index.html

```diff
@@ -71,6 +54,4 @@ <h3>Ofek Manor</h3>
 					I like learning new things, cats and playing video games.</p>
 			</div>
 		</section>
-</body>
-
-</html>
+{%endblock%}
```

Comment:
please add a spaces to make it more readable like:
```suggestion
{% endblock %}
```